test/AsmJs/rlexe.xml

```diff
@@ -566,16 +566,16 @@
   </test>
   <test>
     <default>
-      <files>invalidIntLiteral.js</files>
-      <baseline>invalidIntLiteral.baseline</baseline>
-      <compile-flags>-testtrace:asmjs -simdjs</compile-flags>
+      <files>floatmod.js</files>
+      <baseline>floatmod.baseline</baseline>
+      <compile-flags>-forceserialized -testtrace:asmjs -simdjs</compile-flags>
     </default>
```

Comment:
This probably looks kind of weird, but original purpose of this test was I wanted a basic negative test for bytecode serialization to sanity check reparse logic in that case.

Issue now is if asm.js compilation fails we disable asm.js for the rest of the parse, and since we don't defer anything when we are doing serialization, asm.js is disabled when parsing second module. This caused annoyance getting the trace in baselines to match up, so I moved our negative serialization test to use a different file.